quarkus-debezium-parent/quarkus-debezium-agroal-parent/runtime/src/main/java/io/quarkus/debezium/agroal/configuration/DatasourceParser.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.quarkus.debezium.agroal.configuration;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.jdbc.JdbcConfiguration;
+
+public class DatasourceParser {
+
+    private static final Logger logger = LoggerFactory.getLogger(DatasourceParser.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(DatasourceParser.class);
```
For consistency, and adjust usages in class.